web/js/data/cmr.js

```diff
@@ -1,15 +1,17 @@
-/**
- * @module wv.data
- */
-var wv = wv || {};
-wv.data = wv.data || {};
-wv.data.cmr = wv.data.cmr || {};
-
-wv.data.cmr.client = wv.data.cmr.client || function (spec) {
+import $ from 'jquery';
+import util from '../util/util';
+import OlPolygon from 'ol/geom/polygon';
```

Comment:
Should this be `OlGeomPolygon` for consistency with the other places you imported it?

Reply:
Good catch!